rtloader/test/rtloader/rtloader_test.go

```diff
@@ -69,6 +69,28 @@ with open(r'%s', 'w') as f:
 	helpers.AssertMemoryUsage(t)
 }
 
+func TestSysExecutableValue(t *testing.T) {
```

Comment:
Nice test 👌

Reply:
👍 it's what found the problem with the Windows `Py_SetProgramName` impl :)